modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -23,10 +23,10 @@ The minimum values for the amount of CPU and memory allocated to each replica of
 
 Anypoint Platform allocates native and heap memory for a deployed application. Heap memory is the portion of the total memory that is made available to the Mule runtime and the application. Heap memory is used for tasks like handling payload.
 
-Both CloudHub and Runtime fabric allocate both types of memory. However, they differ in how the memory allocation of each memory type is calculated.
+Both CloudHub and Runtime Fabric allocate both types of memory. However, they differ in how the memory allocation of each memory type is calculated.
 
-* Runtime Fabric: lists the total memory available for an application. The available heap memory is approximately half of the total memory allocated to an application.
-* CloudHub: describes minimum memory requirements in terms of the heap memory available to an application.
+* Runtime Fabric lists the total memory available for an application. The available heap memory is approximately half of the total memory allocated to an application.
```

Comment:
To improve list-item scannability, let's limit each to one fragment or sentence following the bullet, followed by any additional info in a line following the item:

     o Runtime Fabric lists....application.
        The available heap....

GLOBAL